### PR TITLE
Use recommended `eslint-plugin-import` rule sets

### PR DESCRIPTION
**Pull request by rekmarks**
Extends the recommended `eslint-plugin-import` rule sets per their [readme](https://github.com/benmosher/eslint-plugin-import#typescript) (scroll up for recommended JavaScript config).

Additionally, disables the core ESLint rule [`no-duplicate-imports`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md), since it appears to be a subset of `import/no-duplicates` .

### CODE REVIEW

File: packages/typescript/rules-snapshot.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
   "constructor-super": "off",
   "default-param-last": "off",
   "getter-return": "off",
+  "import/named": "off",
```
Comment (Gudahtt) [inline packages/typescript/rules-snapshot.json:69]: :eyes: This doesn't seem right

Comment (Gudahtt) [reply packages/typescript/rules-snapshot.json:69]: Ah https://github.com/benmosher/eslint-plugin-import/blob/master/config/typescript.js#L25

Comment (rekmarks) [reply packages/typescript/rules-snapshot.json:69]: Au contraire! https://github.com/benmosher/eslint-plugin-import/blob/6c8981d/config/typescript.js#L25-L27

File: packages/base/src/index.js [hunk 2]
```diff
@@ -9,9 +9,13 @@ module.exports = {
...
   },
 
-  plugins: ['import', 'prettier'],
+  plugins: ['prettier'],
 
-  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:prettier/recommended',
+    'plugin:import/recommended',
```
Comment (rekmarks) [inline packages/base/src/index.js:17]: This adds the import plugin.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**